src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -196,7 +196,9 @@ void OnNavigationFailed(object sender, NavigationFailedEventArgs e)
 		private void OnSuspending(object sender, SuspendingEventArgs e)
 		{
 			var deferral = e.SuspendingOperation.GetDeferral();
-			//TODO: Save application state and stop any background activity
+
+			Console.WriteLine($"OnSuspending (Deadline:{e.SuspendingOperation.Deadline})");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5617740)